### PR TITLE
fence_gce: Add user agent to API requests

### PR DESCRIPTION
**Pull request by kj1724**

### CODE REVIEW

**Comment by oalbrigt** [comment]
ok to test

File: agents/gce/fence_gce.py [hunk 1]
```diff
@@ -103,6 +109,13 @@ def replace_api_uri(options, http_request):
 
 def retry_api_execute(options, http_request):
 	replaced_http_request = replace_api_uri(options, http_request)
+	action = ACTION_IDS[options["--action"]] if options["--action"] in ACTION_IDS else 0
+	try:
+		user_agent_header = USER_AGENT % (VERSION, options["--image"], action)
```
Comment (oalbrigt) [inline agents/gce/fence_gce.py:114]: You have to add this parameter to `define_new_opts()` and the `device_opt` array.

Comment (oalbrigt) [reply agents/gce/fence_gce.py:114]: Or I guess you can remove `--` from it, as it's auto-detected in your code below.

Comment (kj1724) [reply agents/gce/fence_gce.py]: Thanks, I removed the dashes.

**Comment by oalbrigt** [comment]
Thanks.

**Pull request merged**